.github/workflows/apt.yml

```diff
@@ -0,0 +1,28 @@
+name: Linux
```

Comment:
Pls set appropriate name.

Reply:
I'd used Apt based Linux and Yum based Linux.

Reply:
Debian and derivatives?

Reply:
except for `name:`, LGTM.